diff --git a/browser/base/content/main-popupset.inc.xhtml b/browser/base/content/main-popupset.inc.xhtml
index 81880ad99c93f7d3ffcc02fb3f4de202cc454fa0..e1cbd5a3dfdf56892bbbaf99d98d333cdb100521 100644
--- a/browser/base/content/main-popupset.inc.xhtml
+++ b/browser/base/content/main-popupset.inc.xhtml
@@ -396,10 +396,10 @@
                   seeing the command event pass by. The observes attribute is
                   here so that the menuitem is still disabled and re-enabled
                   correctly. -->
-    <menuitem oncommand="gCustomizeMode.enter()"
+    <!-- <menuitem oncommand="gCustomizeMode.enter()"
               observes="cmd_CustomizeToolbars"
               class="viewCustomizeToolbar"
-              data-lazy-l10n-id="toolbar-context-menu-view-customize-toolbar-2"/>
+              data-lazy-l10n-id="toolbar-context-menu-view-customize-toolbar-2"/> -->
     <menuseparator contexttype="fullscreen"/>
     <menuitem class="fullscreen-context-autohide"
               contexttype="fullscreen"
