diff --git a/browser/base/content/navigator-toolbox.inc.xhtml b/browser/base/content/navigator-toolbox.inc.xhtml
index 5013cfb685e7aa8b330d0ce2a06769360c643304..d69bf34551b48f7b37c3839c1cfd12c422033eb4 100644
--- a/browser/base/content/navigator-toolbox.inc.xhtml
+++ b/browser/base/content/navigator-toolbox.inc.xhtml
@@ -14,7 +14,7 @@
     <!-- Menu -->
     <toolbar type="menubar" id="toolbar-menubar"
              class="browser-toolbar chromeclass-menubar titlebar-color"
-             customizable="true"
+             customizable="false"
              mode="icons"
              context="toolbar-context-menu">
       <toolbaritem id="menubar-items" align="center">
@@ -29,7 +29,7 @@
     <toolbar id="TabsToolbar"
              class="browser-toolbar titlebar-color"
              fullscreentoolbar="true"
-             customizable="true"
+             customizable="false"
              customizationtarget="TabsToolbar-customization-target"
              mode="icons"
              data-l10n-id="tabs-toolbar"
@@ -38,67 +38,7 @@
 
       <hbox class="titlebar-spacer" type="pre-tabs"/>
 
-      <hbox flex="1" align="end" class="toolbar-items">
-        <toolbartabstop/>
-        <hbox id="TabsToolbar-customization-target" flex="1">
-          <toolbarbutton id="firefox-view-button"
-                         class="toolbarbutton-1 chromeclass-toolbar-additional"
-                         data-l10n-id="toolbar-button-firefox-view"
-                         role="button"
-                         aria-pressed="false"
-                         oncommand="FirefoxViewHandler.openTab(event);"
-                         onmousedown="FirefoxViewHandler.openTab(event);"
-                         cui-areatype="toolbar"
-                         removable="true"/>
-
-          <tabs id="tabbrowser-tabs"
-                is="tabbrowser-tabs"
-                aria-multiselectable="true"
-                setfocus="false"
-                tooltip="tabbrowser-tab-tooltip"
-                stopwatchid="FX_TAB_CLICK_MS">
-            <hbox class="tab-drop-indicator" hidden="true"/>
-# If the name (tabbrowser-arrowscrollbox) or structure of this changes
-# significantly, there is an optimization in
-# DisplayPortUtils::MaybeCreateDisplayPortInFirstScrollFrameEncountered based
-# the current structure that we may want to revisit.
-            <arrowscrollbox id="tabbrowser-arrowscrollbox" orient="horizontal" flex="1" style="min-width: 1px;" clicktoscroll="true" scrolledtostart="true" scrolledtoend="true">
-              <tab is="tabbrowser-tab" class="tabbrowser-tab" selected="true" visuallyselected="true" fadein="true"/>
-              <hbox id="tabbrowser-arrowscrollbox-periphery">
-                <toolbartabstop/>
-                <toolbarbutton id="tabs-newtab-button"
-                               class="toolbarbutton-1"
-                               command="cmd_newNavigatorTab"
-                               onclick="checkForMiddleClick(this, event);"
-                               tooltip="dynamic-shortcut-tooltip"/>
-                <spacer class="closing-tabs-spacer" style="width: 0;"/>
-              </hbox>
-            </arrowscrollbox>
-            <html:span id="tabbrowser-tab-a11y-desc" hidden="true"/>
-          </tabs>
-
-          <toolbarbutton id="new-tab-button"
-                         class="toolbarbutton-1 chromeclass-toolbar-additional"
-                         data-l10n-id="tabs-toolbar-new-tab"
-                         command="cmd_newNavigatorTab"
-                         onclick="checkForMiddleClick(this, event);"
-                         tooltip="dynamic-shortcut-tooltip"
-                         ondrop="newTabButtonObserver.onDrop(event)"
-                         ondragover="newTabButtonObserver.onDragOver(event)"
-                         ondragenter="newTabButtonObserver.onDragOver(event)"
-                         cui-areatype="toolbar"
-                         removable="true"/>
-
-          <toolbarbutton id="alltabs-button"
-                         class="toolbarbutton-1 chromeclass-toolbar-additional tabs-alltabs-button"
-                         delegatesanchor="true"
-                         badged="true"
-                         onkeypress="gTabsPanel.showAllTabsPanel(event, 'alltabs-button');"
-                         onmousedown="gTabsPanel.showAllTabsPanel(event, 'alltabs-button');"
-                         data-l10n-id="tabs-toolbar-list-all-tabs"
-                         removable="false"/>
-        </hbox>
-      </hbox>
+      
 
       <hbox class="titlebar-spacer" type="post-tabs"/>
 
@@ -108,7 +48,7 @@
         <label data-l10n-id="private-browsing-indicator-label"></label>
       </hbox>
 
-#include titlebar-items.inc.xhtml
+    #include titlebar-items.inc.xhtml
 
     </toolbar>
 
@@ -117,7 +57,7 @@
   <toolbar id="nav-bar"
            class="browser-toolbar"
            data-l10n-id="navbar-accessible"
-           fullscreentoolbar="true" mode="icons" customizable="true"
+           fullscreentoolbar="true" mode="icons" customizable="false"
            customizationtarget="nav-bar-customization-target"
            overflowable="true"
            default-overflowbutton="nav-bar-overflow-button"
@@ -407,7 +347,8 @@
         <toolbartabstop/>
       </toolbaritem>
 
-      <toolbarspring cui-areatype="toolbar" class="chromeclass-toolbar-additional"/>
+      <!-- After URL BAR -->
+      <!-- <toolbarspring cui-areatype="toolbar" class="chromeclass-toolbar-additional"/>
 
       <toolbarbutton id="downloads-button"
                      class="toolbarbutton-1 chromeclass-toolbar-additional"
@@ -499,10 +440,31 @@
                      badged="true"
                      consumeanchor="PanelUI-button"
                      data-l10n-id="appmenu-menu-button-closed2"/>
-    </toolbaritem>
+    </toolbaritem> -->
+    </hbox>
+    <hbox class="titlebar-buttonbox-container" skipintoolbarset="true">
+      <hbox class="titlebar-buttonbox titlebar-color">
+        <toolbarbutton class="titlebar-button titlebar-min"
+                       oncommand="window.minimize();"
+                       data-l10n-id="browser-window-minimize-button"
+                       />
+        <toolbarbutton class="titlebar-button titlebar-max"
+                       oncommand="window.maximize();"
+                       data-l10n-id="browser-window-maximize-button"
+                       />
+        <toolbarbutton class="titlebar-button titlebar-restore"
+                       oncommand="window.fullScreen ? BrowserFullScreen() : window.restore();"
+                       data-l10n-id="browser-window-restore-down-button"
+                       />
+        <toolbarbutton class="titlebar-button titlebar-close"
+                       command="cmd_closeWindow"
+                       data-l10n-id="browser-window-close-button"
+                       />
+      </hbox>
+    </hbox>
   </toolbar>
 
-  <toolbar id="PersonalToolbar"
+  <!-- <toolbar id="PersonalToolbar"
            mode="icons"
            class="browser-toolbar chromeclass-directories"
            context="toolbar-context-menu"
@@ -565,7 +527,7 @@
         </toolbarbutton>
       </hbox>
     </toolbaritem>
-  </toolbar>
+  </toolbar> -->
 
   <html:template id="tab-notification-deck-template">
     <html:named-deck id="tab-notification-deck"></html:named-deck>
