diff --git a/browser/themes/shared/jar.inc.mn b/browser/themes/shared/jar.inc.mn
index 7a1dfd61ac864af5cdcd0d7690f57c2cf0921941..021cc22f98d6f9f66ee6a9ee786bed744054f8bc 100644
--- a/browser/themes/shared/jar.inc.mn
+++ b/browser/themes/shared/jar.inc.mn
@@ -16,7 +16,7 @@
   skin/classic/browser/addon-notification.css                  (../shared/addon-notification.css)
   skin/classic/browser/autocomplete.css                        (../shared/autocomplete.css)
   skin/classic/browser/blockedSite.css                         (../shared/blockedSite.css)
-  skin/classic/browser/browser-shared.css                      (../shared/browser-shared.css)
+* skin/classic/browser/browser-shared.css                      (../shared/browser-shared.css)
   skin/classic/browser/ctrlTab.css                             (../shared/ctrlTab.css)
   skin/classic/browser/identity-credential-notification.css    (../shared/identity-credential-notification.css)
   skin/classic/browser/light-dark-overrides.css                (../shared/light-dark-overrides.css)
